syntax = "proto3";

service SimpleServer {
  rpc Command (CommandRequest) returns (CommandReply) {}
}

message CommandRequest {
  string args = 1;
  enum CommandType {
    kEcho = 0;
    kQuit = 1;
  }
  CommandType type = 2;;
}

message CommandReply {
  bool return_code = 1;
  string message = 2;
}

